src/tools/nvmf-rbd-gw/nvmf-rbd-gw.c

```diff
@@ -0,0 +1,964 @@
+/*-
+ *   BSD LICENSE
```

Comment:
Nit: new license type for Ceph code?

Reply:
This file is modified copy from spdk/examples/nvmf/nvmf.c, which originally has bsd license. So I kept as is.

Reply:
Ack -- will just need someone w/ more knowledge about the implications to weigh in here prior to merging.